### PR TITLE
LLT-5253: Bump libtelio-build tag

### PR DESCRIPTION
**Pull request by lcruz99**
Bump `libtelio-build` tag.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Comment by olekoliinyk** [comment]
+ 1.0

**Review by olekoliinyk — APPROVED** [review]

File: .unreleased/LLT-5253 [hunk 1]
```diff
@@ -0,0 +1 @@
+ci: Bump libtelio-build tag
```
Comment (jjanowsk) [inline .unreleased/LLT-5253:1]: do we really want to see it in the changelog? I guess that's the perfect example of a PR with an empty changelog file

Comment (lcruz99) [reply .unreleased/LLT-5253]: oh, doesn't make sense then. cleaned that

**Review by jjanowsk — DISMISSED** [review]

**Review by matislovas — APPROVED** [review]
+2

**Review by jjanowsk — APPROVED** [review]

**Comment by lcruz99** [comment]
[CLOSED] No longer relevant, tag is already bumped.

**Pull request closed without merging**